files/test-app-overrides/config/ember-try.js

```diff
@@ -48,24 +48,6 @@ module.exports = async function () {
           },
         },
       },
-      {
-        name: 'ember-classic',
```

Comment:
Removed from the v1 addon blueprint as well.

Reply:
Thank you!